BREAKING_CHANGES.md

```diff
@@ -29,6 +32,26 @@
 <!-- docker run --rm -it -v $(pwd):/usr/src jorgeandrada/doctoc --github -->
 
 
+# 7.8.1 - 2020/07/28
+
+### Add headless Service for StatefulSet
+
+A headless `Service` has been added to Logstash chart in [#695][].
+
+Headless `Service` is required for `Statefulsets`. While Helm 2 allowed to
+deploy `Statefulset` without a `serviceName`, however Helm 3 enforce this
+requirement and fails if `serviceName` is missing.
+
+`Statefulset` don't accept `serviceName` field update during release upgrades.
+Upgrading Logstash chart from a previous version will require using
+`helm upgrade --force`.
+
```

Comment:
Minor grammar tweaks for these last 2 paragraphs

```
The headless `Service` is required for `Statefulsets`. Helm 2 allowed
deploying a `Statefulset` without a `serviceName`, however Helm 3 enforces this
requirement and fails if `serviceName` is missing.

`Statefulset` does not accept the `serviceName` field update during release upgrades.
Upgrading the Logstash chart from a previous version will require using
`helm upgrade --force`.
```

Reply:
updated in 2fbfcf0299a794b2bb35db04a38da27a064e429f